assets/js/modules/analytics-4/datastore/report.js

```diff
@@ -370,6 +370,34 @@ const baseSelectors = {
 
 		return isZeroReport( report );
 	} ),
+
+	/**
+	 * Gets a given report for each of the provided audiences.
+	 *
+	 * NOTE: This will be refactored to use pivot reports in #8484.
```

Comment:
@zutigrm I use `TODO:` which get's hightlighted in VSCode for me because of a plugin I have, should it be `@TODO`? Looking at the app we use a mixture of `@TODO` and `TODO:`. I couldn't find a reference to this in the Handbook.

Reply:
I usually use @TODO, to indicate annotation, although it is more of a thing of personal preference. We don;t have abything about it in handbook, this is purely to make it easier in the future to identify parts that need some re-work, as simple search for 'todo' term would point us to these lines